src/posts/entities/post.repository.ts

```diff
@@ -56,9 +63,9 @@ export class PostRepository extends Repository<Post> {
    */
   public async flagPostCreation(flag: boolean, post: Post): Promise<void> {
     post.created = flag;
-    // for now add ready = true whenever flag = true
-    // this should be changed later whenever we have implementation for media in post
-    if (flag) {
+
+    // make post ready if it has no media or all media got handled
```

Comment:
provide better comment for others by mentioning that this is related to making post ready if it doesn't have media, for example `text_poll` or `mini_survey` without media.